scripts/tkg_byoi.py

```diff
@@ -351,12 +359,17 @@ def render_folder_and_append(folder):
     applies the Jinja2 templating using jinja_args_map dictionary.
     """
     output = {}
+    print("Listing files to render and append:")
```

Comment:
Did you miss to log some variable here? Rest of the code in this function is not logging anything.

Reply:
ITs just the action that is about to happen and not listing as in printing